src/Validator/Sequentially.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+/*
+ * This file was created by developers working at BitBag
+ * Do you need more information about us and what we do? Visit our https://bitbag.io website!
+ * We are hiring developers from all over the world. Join us and start your new, exciting adventure and become part of us: https://bitbag.io/career
+ */
+
+declare(strict_types=1);
+
+namespace BitBag\SyliusProductBundlePlugin\Validator;
+
+use Symfony\Component\Validator\Constraints\Composite;
+
+final class Sequentially extends Composite
```

Comment:
What does it do?

Reply:
@er1z provides a compatibility with Symfony 4.4. Sequentially constraint has been introduced in Symfony 5.x.